src/main/java/com/opams/arch/entities/ProductEntity.java

```diff
@@ -1,8 +1,38 @@
 package com.opams.arch.entities;
 
-import javax.persistence.Entity;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.math.BigDecimal;
 
 @Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
 public class ProductEntity {
+
+    @GeneratedValue
+    @Id
```

Comment:
Which kind of strategy you want to use for id?
AUTO? SEQUENCE? TABLE? IDENTITY?

and WHY?